requirements.txt

```diff
@@ -12,4 +12,6 @@ python_dateutil >= 2.5.3
 pytest~=6.1.1
 pylint~=2.6.0
 pytest-pylint~=0.18.0
-bump2version~=1.0.1
\ No newline at end of file
+bump2version~=1.0.1
+gitpython~=3.1.11
```

Comment:
Ok, this gets better with emptying ``~/.docker`` folder completely. However, the following arises, with reference to ``tests/integration/X-Road/src/systemtest/op-monitoring/misc/test_bulk_operational_data.py``:

```
python -m pytest \
	-v \
	--pylint --pylint-rcfile=setup.cfg \
	--cov=xrdsst/controllers \
	--cov-report=term \
	--cov-report=html:coverage-report \
	tests\

============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-6.1.2, py-1.9.0, pluggy-0.13.1 -- /home/uns/dev/niis/xrdsst-review/X-Road-Security-Server-toolkit/env/bin/python
cachedir: .pytest_cache
rootdir: /home/uns/dev/niis/xrdsst-review/X-Road-Security-Server-toolkit
plugins: pylint-0.18.0, cov-2.10.1
collecting ... collected 69 items / 1 error / 68 selected
--------------------------------------------------------------------------------
Linting files
.......................
--------------------------------------------------------------------------------

==================================== ERRORS ====================================
_ ERROR collecting tests/integration/X-Road/src/systemtest/op-monitoring/misc/test_bulk_operational_data.py _
env/lib/python3.8/site-packages/urllib3/connection.py:159: in _new_conn
    conn = connection.create_connection(
env/lib/python3.8/site-packages/urllib3/util/connection.py:84: in create_connection
    raise err
env/lib/python3.8/site-packages/urllib3/util/connection.py:74: in create_connection
    sock.connect(sa)
E   ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:
env/lib/python3.8/site-packages/urllib3/connectionpool.py:665: in urlopen
    httplib_response = self._make_request(
env/lib/python3.8/site-packages/urllib3/connectionpool.py:387: in _make_request
    conn.request(method, url, **httplib_request_kw)
/usr/lib/python3.8/http/client.py:1255: in request
    self._send_request(method, url, body, headers, encode_chunked)
/usr/lib/python3.8/http/client.py:1301: in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
/usr/lib/python3.8/http/client.py:1250: in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
/usr/lib/python3.8/http/client.py:1010: in _send_output
    self.send(msg)
/usr/lib/python3.8/http/client.py:950: in send
    self.connect()
env/lib/python3.8/site-packages/urllib3/connection.py:187: in connect
    conn = self._new_conn()
env/lib/python3.8/site-packages/urllib3/connection.py:171: in _new_conn
    raise NewConnectionError(
E   urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7fe50373a580>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:
env/lib/python3.8/site-packages/requests/adapters.py:439: in send
    resp = conn.urlopen(
env/lib/python3.8/site-packages/urllib3/connectionpool.py:719: in urlopen
    retries = retries.increment(
env/lib/python3.8/site-packages/urllib3/util/retry.py:436: in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
E   urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=2080): Max retries exceeded with url: /store_data (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe50373a580>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:
tests/integration/X-Road/src/systemtest/op-monitoring/misc/test_bulk_operational_data.py:80: in <module>
    response = requests.post(
env/lib/python3.8/site-packages/requests/api.py:116: in post
    return request('post', url, data=data, json=json, **kwargs)
env/lib/python3.8/site-packages/requests/api.py:60: in request
    return session.request(method=method, url=url, **kwargs)
env/lib/python3.8/site-packages/requests/sessions.py:533: in request
    resp = self.send(prep, **send_kwargs)
env/lib/python3.8/site-packages/requests/sessions.py:646: in send
    r = adapter.send(request, **kwargs)
env/lib/python3.8/site-packages/requests/adapters.py:516: in send
    raise ConnectionError(e, request=request)
E   requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=2080): Max retries exceeded with url: /store_data (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe50373a580>: Failed to establish a new connection: [Errno 111] Connection refused'))
```

Reply:
Above was caused by `tearDown` not applying after completely failed run to clean up ``tests/integration/X-Road`` folder. It could be added to ``clean`` target of Makefile most likely as precondition for ``test-all``.